content/docs/release-notes/2305-0-release-notes.md

```diff
@@ -0,0 +1,196 @@
+---
+linktitle: 23.05.0 Release Notes
+title: Release Notes for Open 3D Engine 23.05.0
+description: Full release notes for Open 3D Engine (O3DE) version 23.05.0.
+weight: 895
+toc: true
+---
+
+
+**Open 3D Engine (O3DE)** 23.05 release levels up the authoring experience for game creators and robotics simulation developers. In addition to improvements to the core engine, robotics users in O3DE now have the ROS2 Gem, a Gem that provides integration with the [Robot Operating System (ROS2)](https://www.ros.org/), which provides drivers, state-of-the-art algorithms, and developer tools for developing robotics simulations. Game developers in O3DE now have the [Multiplayer Sample game](https://github.com/o3de/o3de-multiplayersample) available to them - a ready-to-use game that's not only fun to play, but provides developers with example implementations of common game elements to help start their own projects.
```

Comment:
@chanmosq 
We determined we might want to be clearer about ROS2. There's discussion that starts here https://discord.com/channels/805939474655346758/816043864858951710/1100868315372605530 

Read until Adam's comment where he mentions "I would consider the previous version "early-access".
. ROS2 isn't new in this release, but it's a significant update.  Not sure if you want to change the wording here. When I originally wrote the above , I wrote it with "ROS 2 is new" in mind.